### PR TITLE
doc: add doc of setting up your pipeline

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

part of #493

### CODE REVIEW

File: docs/content/en/docs/pipeline/setting.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+To start using Kurator pipeline, you need to install Tekton in your Kubernetes cluster. Run the following command to install Tekton components:
+
+```
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/triggers/latest/release.yaml
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/triggers/latest/interceptors.yaml
+kubectl apply --filename https://storage.googleapis.com/tekton-releases/chains/latest/release.yaml
+```
+
+### Configuring Authentication for Private Repositories
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/pipeline/setting.md:24]: nit: how can we make it optional

**Review by hzxuzhonghu — APPROVED** [review]
Do you plan to add some other complex examples in other doc

**Comment by Xieql** [comment]
> Do you plan to add some other complex examples in other doc

this is just a doc for concept(Subdocument 1: Setting Up Your Pipeline), there stil need two doc pr about:

- Subdocument 2: Practical Operation Guide

- Subdocument 3: Supply Chain Security

**Review by hzxuzhonghu — COMMENTED** [review]
OK
/lgtm

**Pull request merged**